### PR TITLE
Issue #37: Result.value

### LINKED ISSUE #37: Result.value
I think it might be beneficial to include a `Result.value()` function similar to the `Maybe.value()` function, either returning the `Ok` or `Err` contained. Currently adding this in a fork and will open a PR once complete

### PR DESCRIPTION
**Pull request by JackSpagnoli**
Adds Result.value function to return the value in the Ok or Err.
Resolves #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,26 +30,28 @@ Table of Contents
...
       + [.swap](#resultswap)
+      + [.bimap](#resultbimap)
+      + [.ap](#resultap)
+      + [.inspect](#resultinspect)
       + [.cata](#resultcata)
       + [.toMaybe](#resulttomaybe)
-      + [.inspect](#resultinspect)
-      + [.isErr](#resultiserr)
       + [.isOk](#resultisok)
+      + [.isErr](#resultiserr)
```
Comment (rametta) [inline README.md:53]: Seems like this file has a lot of lines touched, can you update to just add the 1 line needed?

File: specs/result.spec.ts [hunk 2]
```diff
@@ -217,4 +217,10 @@ describe('Result', () => {
     done()
   })
 
+  it('should return the value or error', () => {
+    expect(Ok(6).value()).toBe(6)
+
+    expect(Err(new Error("Test Error")).value()).toEqual(new Error("Test Error"))
```
Comment (rametta) [inline specs/result.spec.ts:223]: Let's just use a string to make sure we're not doing an equality check on a full object which may not give desired effects
```suggestion
    expect(Err("Test Error").value()).toEqual("Test Error")
```

File: README.md [hunk 3]
```diff
@@ -475,6 +477,19 @@ log(isOver6Feet(7).isErr()) // false
...
 ```
 
+##### Result.value
+Returns either the value contained in the Ok, or the error in the Err
+
+```js
+import { Ok, Err } from 'pratica'
+
+const six = Ok(6).value()
+const error = Err(new Error("Test Error")).value()
```
Comment (rametta) [inline README.md:487]: Let's not use new Error in the examples, could get confusing

**Review by rametta — COMMENTED** [review]
Thanks for the PR, a few minor comments

File: README.md (hunk 1 above)
Comment (JackSpagnoli) [reply README.md:53]: I noticed that the contents list was out of order and missing a link to Maybe.value. the fixes for that make up most of the lines. I'll remove them if you like

File: specs/result.spec.ts (hunk 2 above)
Comment (JackSpagnoli) [reply specs/result.spec.ts]: Fixed in commit e8f5400f54471f7d1a9d2ebbdbe553aad6fced65

File: README.md (hunk 3 above)
Comment (JackSpagnoli) [reply README.md]: Fixed in commit e8f5400f54471f7d1a9d2ebbdbe553aad6fced65

File: README.md (hunk 1 above)
Comment (JackSpagnoli) [reply README.md:51]: Resolved by commit cdf5ec64abb0469bd2b53ab2657eae3b31630156

**Review by rametta — APPROVED** [review]

**Comment by rametta** [comment]
Completely forgot about this. Thanks for the PR!

**Pull request merged**